cmd/juju/common/flags.go

```diff
@@ -34,8 +34,12 @@ func (f *ConfigFlag) Set(s string) error {
 		return nil
 	}
 	var value interface{}
-	if err := yaml.Unmarshal([]byte(fields[1]), &value); err != nil {
-		return errors.Trace(err)
+	if fields[1] == "" {
+		value = fields[1]
```

Comment:
seeing as you've already assert that it's "", I think it would be clearer if you made this `value = ""`